build/media-suite_helper.sh

```diff
@@ -425,15 +425,16 @@ do_extract() {
     local nocd="${nocd:-}"
     local archive="$1" dirName="$2"
     # accepted: zip, 7z, tar, tar.gz, tar.bz2 and tar.xz
-    [[ -z "$dirName" ]] && dirName=$(guess_dirname "$archive")
-    if [[ $dirName != "." && -d "$dirName" ]] &&
-        { [[ $build32 = "yes" ]] && { [[ ! -f "$dirName"/build_successful32bit ]] ||
-            [[ -n "$flavor" && ! -f "$dirName/build_successful32bit_${flavor}" ]]; } ||
-          [[ $build64 = "yes" ]] && { [[ ! -f "$dirName"/build_successful64bit ]] ||
-            [[ -n "$flavor" && ! -f "$dirName/build_successful64bit_${flavor}" ]]; };
+    [[ -z $dirName ]] && dirName=$(guess_dirname "$archive")
+    if [[ $dirName != "." && -d $dirName ]] &&
+        {
+            [[ $build32 == "yes" ]] && { [[ ! -f "$dirName"/build_successful32bit ]] ||
+                [[ -n $flavor && ! -f "$dirName/build_successful32bit_${flavor}" ]]; } ||
+                [[ $build64 == "yes" ]] && { [[ ! -f "$dirName"/build_successful64bit ]] ||
```

Comment:
This doesn't seem right. Isn't this line supposed to have one less indent?

Reply:
I think it indented every time it hit `||`